tests/lib/rules/space-before-function-paren.js

```diff
@@ -94,7 +94,18 @@ ruleTester.run("space-before-function-paren", rule, {
         },
         { code: "var bar = function foo () {}",
           options: [ { named: "ignore", anonymous: "always" } ]
-        }
+        },
+
+        // Async arrow functions
+        { code: "() => 1", parserOptions: {ecmaVersion: 6} },
+        { code: "async a => a", parserOptions: {ecmaVersion: 8} },
```

Comment:
To prevent someone missing this case if the code changes in the future, can you add a test with this same code but the "never" option?